nilearn/decoding/decoder.py

```diff
@@ -253,8 +248,10 @@ class _BaseDecoder(LinearModel, RegressorMixin, CacheMixin):
         for clustering_percentile equal to 10.
 
     screening_percentile: int, float, optional, in the closed interval [0, 100]
-        Perform a univariate feature selection based on the Anova F-value for
-        the input data. A float according to a percentile of the highest
+        The percentage of brain volume that will be kept with respect to a full
+        MNI template. In particular, if it is lower than 100, a univariate
```

Comment:
(Actually it will be performed quite systematically, given that the MNI template is rather large... but that's a detail I think)

Reply:
So do you think the description is good for this parameter? Does it need a bit more detail?

Reply:
That's fine for the moment.